src/config/mod.rs

```diff
@@ -111,11 +111,11 @@ macro_rules! set_from_flag {
 
 impl Config {
 
-    pub fn default() -> Config {
+    pub fn play_out_aftermath(play_out_aftermath: bool) -> Config {
```

Comment:
But we're modifying it during the command line parsing, don't we? So we could just create a `mut Config` in the test setup, change the option and the wrap it in an `Arc`. At least I assumed that this would work as we seem to be doing exactly that in `main.rs`.